app/controllers/test/oidc_test_controller.rb

```diff
@@ -84,7 +84,7 @@ def acr_values(ial:, aal:)
         '' => 'http://idmanagement.gov/ns/assurance/ial/1',
         '1' => 'http://idmanagement.gov/ns/assurance/ial/1',
         '2' => 'http://idmanagement.gov/ns/assurance/ial/2',
-        'biometric-comparison-required' => 'http://idmanagement.gov/ns/assurance/ial/2',
+        'facial-match-required' => 'http://idmanagement.gov/ns/assurance/ial/2',
```

Comment:
this value doesn't matter because the `ial2_params` method adds the values. (it currently adds a vector of trust, but i will be updating that soon)